app/src/main/java/org/sopt/at/feature/onboarding/SignUpStep.kt

```diff
@@ -0,0 +1,6 @@
+package org.sopt.at.feature.onboarding
+
+enum class SignUpStep {
+    ID,
+    PASSWORD
```

Comment:
```suggestion
    ID,
    PASSWORD,
```

확장 가능성이 있는 부분에는 끝부분에 반점(트레일링 콤마) 찍어주면 좋습니다.

Reply:
오호 작은 습관도 챙겨야겠슴니다